wiki/scripts/build.sh

```diff
@@ -43,11 +43,13 @@ rebuild() {
 
```

Comment:
You are looking at doc for `an` older version ...

Do we show this warning on master? We shouldn't. We could have a different warning that you are looking at docs for `master` branch. The latest released version is ...